app/models/concerns/itemizable.rb

```diff
@@ -130,6 +130,19 @@ def line_item_items_quantity_is_positive
     end
   end
 
+  def line_item_items_quantity_is_at_least(threshold)
```

Comment:
Is there a reason `items_` is repeated here (and in similar methods)?

Reply:
@dorner that is how the method was created originally and I did not alter the title. I can edit if you would like!

Reply:
I do not mind at all, Ill get right on it :)